inc/events-calendar.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+/**
+ * Add Sidebar to events section
+ *
+ */
+
+
+function nightingale_widgets_events() {
+	register_sidebar(
+		array(
+			'name'          => esc_html__( 'Events Sidebar', 'nightingale' ),
+			'id'            => 'events-side',
+			'description'   => esc_html__( 'Elements to show in the page sidebar. each widget will show as a panel. If empty you will have a blank right hand panel.', 'nightingale' ),
+			'before_widget' => '<section id="%1$s" class="nhsuk-related-nav %2$s">',
+			'after_widget'  => '</section>',
+			'before_title'  => '<h2 class="nhsuk-related-nav__heading">',
+			'after_title'   => '</h2>',
+		)
+	);
+}
+
+add_action( 'widgets_init', 'nightingale_widgets_events', 10 );
+
+
+
+/**
+ * Remove end time of event on event archive page
+ *
+ * @link https://gist.github.com/b76421f2490a8b8995493f203e11b331
+ *
+ * @param array $formatting_details
+ *
+ * @return array
+ */
+
+function tribe_remove_end_time_single( $formatting_details ) {
+	$formatting_details['show_end_time'] = 0;
+
+	return $formatting_details;
+}
+add_filter( 'tribe_events_event_schedule_details_formatting', 'tribe_remove_end_time_single', 10, 2);
+
+
+
+
+/**
+ * Location of Calendar Events folder so can change easily.
+ */
+
+function nightingale_events_template_location(){
+	return 'tribe/events/';
+}
+
+
+
+
+/**
+ * The Events Calendar and related plugins: Add your own location for template file loading.
+ *
+ * @link https://gist.github.com/b76421f2490a8b8995493f203e11b331
+ *
+ * @see \Tribe__Events__Templates::getTemplateHierarchy()
+ *
+ * @param string $file     The full file path trying to be loaded.
+ * @param string $template The template name, such as
+ *
+ * @return string
+ */
+
+function tribe_additional_template_locations( $file, $template ) {
```

Comment:
does this function need to be tribe_ prefix? If its unique to the theme, it should be a nightingale_ prefix.